lucetc/src/lib.rs

```diff
@@ -248,13 +256,26 @@ impl<T: AsLucetc> LucetcOpts for T {
     }
 
     fn count_instructions(&mut self, count_instructions: bool) {
-        self.as_lucetc().count_instructions = count_instructions;
+        self.as_lucetc()
+            .builder
+            .count_instructions(count_instructions);
     }
 
     fn with_count_instructions(mut self, count_instructions: bool) -> Self {
         self.count_instructions(count_instructions);
         self
     }
+
+    fn canonicalize_nans(&mut self, canonicalize_nans: bool) {
+        self.as_lucetc()
+            .builder
+            .canonicalize_nans(canonicalize_nans);
+    }
+
+    fn with_canonicalize_nans(mut self, canonicalize_nans: bool) -> Self {
+        self.canonicalize_nans(canonicalize_nans);
+        self
+    }
```

Comment:
Wonderful touch adding these as well :heart:  Thank you!

Reply:
Heads up that you will see a merge conflict here<sup>[1](https://github.com/bytecodealliance/lucet/pull/423/files#diff-a5ee6b9e6496674851a8b1023f579e5eR261)</sup>, now that #423 has landed.

1: https://github.com/bytecodealliance/lucet/pull/423/files#diff-a5ee6b9e6496674851a8b1023f579e5eR261